### ISSUE TITLE
SLGOF not robust with sgcr

### ISSUE THREAD

**Issue by berland** [equinor/pyscal#16]
`hypothesis` managed to find a corner case where at least SLGOF output is invalid:
```
sgcr = 0.04995000000000013
wog = WaterOilGas(swl=0, sorg=0, sgcr=sgcr, h=0.05)
wog.gasoil.add_corey_gas(krgmax=1)
wog.gasoil.add_corey_oil()
```
First data point in SLGOF should be zero, but it is not. Probably related to 0.00005 and 0.00000 being regarded as equal with the default setting of SWINTEGERS.

**Comment by berland** [comment]
If `sgcr = 0.495` then we are good again.

**Comment by berland** [comment]
Output when bug is triggered:
```
print(wog.gasoil.SLGOF())
SLGOF
-- 
-- Sl Krg Krog Pc
-- swirr=0, sgcr=0.04995, swl=0, sorg=0
-- Corey krg, ng=2, krgend=1, krgmax=1
-- Corey krog, nog=2, kroend=1
-- krg = krog @ sw=0.51278
-- Zero capillary pressure
0.0000500 0.9998947 0.0000000 0
0.0500500 0.8974125 0.0025050 0
0.1000500 0.8004698 0.0100100 0
0.1500500 0.7090666 0.0225150 0
0.2000500 0.6232031 0.0400200 0
0.2500500 0.5428791 0.0625250 0
0.3000500 0.4680948 0.0900300 0
0.3500500 0.3988500 0.1225350 0
0.4000500 0.3351448 0.1600400 0
0.4500500 0.2769792 0.2025450 0
0.5000500 0.2243531 0.2500500 0
0.5500500 0.1772667 0.3025550 0
0.6000500 0.1357198 0.3600600 0
0.6500500 0.0997125 0.4225650 0
0.7000500 0.0692448 0.4900700 0
0.7500500 0.0443167 0.5625750 0
0.8000500 0.0249281 0.6400800 0
0.8500500 0.0110792 0.7225850 0
0.9000500 0.0027698 0.8100900 0
0.9500500 0.0000000 0.9025950 0
1.0000000 0.0000000 1.0000000 0
/
```

**Comment by berland** [comment]
Solved in #17
